dlk/python/dlk/core/graph_pattern_matching.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""Graph pattern matching module."""
+
```

Comment:
Why All methods and class don't have docstring in `graph_pattern_matching.py`?

Reply:
Which methods and class is public to another module ?

Reply:
Need to add documentation. Thank you.